iosApp/PhongKMMIC/Sources/Presentation/Modules/Splash/SplashView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SplashView.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 11/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct SplashView: View {
+
+    @State private var isLoaded = false
+
+    var body: some View {
+        GeometryReader { geometry in
```

Comment:
We can omit this `GeometryReader` here 🙏

Reply:
@markgravity, would you fill me in? Why can we omit `GeometryReader`? 🤔

Reply:
@phongvhd93 You can check my approach [here](https://github.com/markgravity/kmm-ic/pull/56), mostly the same as you 😁